content/en/tutorials/lotus/backfill-messages.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Backfilling Messages in Lotus"
+description: "This tutorial guides you through the process of backfilling messages in Lotus."
+lead: "This tutorial guides you through the process of backfilling messages in Lotus for versions v1.25.0 or higher, ensuring your Lotus node is up-to-date with the blockchain's history."
+draft: false
+menu:
+    tutorials:
+        parent: "tutorials-lotus"
+weight: 235
+toc: true
+---
+
+## Overview
+This tutorial guides you through the process of backfilling messages in Lotus for versions v1.25.0 or higher, ensuring your Lotus node is up-to-date with the blockchain's history.
+
+## Prerequisites
+Lotus version 1.25.0 or higher installed.
+Basic understanding of Lotus configuration and command-line tools.
+
+## Configuration Settings
+Update Lotus `config.toml`: Ensure your `config.toml` file includes the following settings:
+
+```toml
+EnableEthRPC = true
+EthTxHashMappingLifetimeDays = 0
+DisableRealTimeFilterAPI = false
+DisableHistoricFilterAPI = false
+EnableMsgIndex = true
+```
+
+## Backfill Process
+### Step 1: Verify or Adjust Configuration
+Ensure the `config.toml` settings are correctly set as described above.
+
+### Step 2: Initiate Backfill
+Run the backfill command via `lotus-shed`. For a comprehensive backfill, use a high epoch number:
+
+```shell
+lotus-shed indexes backfill-msgindex --epochs=9999999
+```
+In the output of the backfill command, the `nrRowsAffected` metric indicates how many messages have been backfilled. A value of 0 means no new messages were backfilled, as they already exist in the database.
+
+By default, `backfill-msgindex` starts from the current head and processes back through the chain for the specified number of epochs. If necessary, you can stop the backfill process and restart it from an earlier point in the chain by using the `--from option` to ensure thorough coverage. This can be verified if `nrRowsAffected` remains 0, indicating no missed messages.
+
+{{< alert icon="tip" >}}
+Optional Database Location Override: For complex setups, use `--repo=/path/to/repo` to specify the database location
+{{< /alert >}}
+
```

Comment:
I think we also should add an alert here that says:
"
Leaving it running beyond FEVM, epoch xxxxxx, isn't useful. 
"

So people do not let it run for many days beyond necessairy.